modules/base/rendering/renderableplane.cpp

```diff
@@ -78,6 +79,31 @@ namespace {
         "aspect ratio of the content. Otherwise it will remain in the given size."
     };
 
+    constexpr openspace::properties::Property::PropertyInfo ScaleByDistanceInfo = {
+        "ScaleByDistance",
+        "Scale By Distance",
+        "Decides whether the plane should automatically adjust in size to match the "
+        "distance to the camera. Otherwise it will remain in the given size."
```

Comment:
```suggestion
        "Decides whether the plane should automatically adjust in size to based on "
        "the distance to the camera. Otherwise it will remain in the given size."
```

Since it doesn't match the distance of the camera.